src/mono/browser/runtime/loader/config.ts

```diff
@@ -324,6 +324,11 @@ async function loadBootConfig (module: DotnetModuleInternal): Promise<void> {
         }
     }
 
+    // Prefer user-defined application environment
```

Comment:
What's the motivation for this exception of the general order of precedence ?

Reply:
It's part of Blazor's public API. I have added a todo to the tracking issue to come back to it